spidermon/contrib/scrapy/monitors.py

```diff
@@ -87,23 +88,60 @@ def test_should_finish_with_expected_reason(self):
 class UnwantedHTTPCodesMonitor(BaseScrapyMonitor):
     """Check for maximum number of unwanted HTTP codes.
 
-    You can configure a ``dict`` of unwanted HTTP codes with
+    Usage # 1
```

Comment:
The documentation for this monitor is generated automatically from the docstrings, but I'd suggest that you build the docs and check if they look good. Install the [docs requirements](https://github.com/scrapinghub/spidermon/blob/master/requirements-docs.txt) in your project, and [build the docs from this directory](https://github.com/scrapinghub/spidermon/tree/master/docs) using `make html`.

Reply:
```
Traceback (most recent call last):
  File "Documents/pyenv/spidermon/bin/sphinx-build", line 10, in <module>
    sys.exit(main())
  File "Documents/pyenv/spidermon/lib/python3.6/site-packages/sphinx/cmd/build.py", line 313, in main
    locale.setlocale(locale.LC_ALL, '')
  File "Documents/pyenv/spidermon/lib/python3.6/locale.py", line 598, in setlocale
    return _setlocale(category, locale)
locale.Error: unsupported locale setting
```
Getting error on the following command.

Reply:
Documentation verified.